<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Shilpa Nadimpalli Kobren</title>

    <script src="jquery-1.11.2.min.js"></script>

    <!-- Bootstrap Core CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">

    <!-- jQuery -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.min.js"></script>
    <script src="jquery.jtruncate.js"></script>
    
    <script type="text/javascript">
      $().ready(function() {  
         $('.truncate').jTruncate();  
       });  
    </script>
     
    
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

    <style type="text/css">
    .title {font-weight:bold; font-size:1.15em; margin:0; padding:0;}
    .author {font-size:.9em; margin:0;}
    .journal {font-style:italic; margin:0;}
    .abstract {font-size:.8em; margin:0 0 1% 0;}
    .header {font-weight:bold; font-size:1.3em; margin:0 0 10px 0; padding:0;}
    ul {padding-left:1em; font-size:.85em;}
    a img {border:1px solid #DDD;}
    a img:hover {border:1px solid #AAA; }
    </style>

    <link href='http://fonts.googleapis.com/css?family=Handlee' rel='stylesheet' type='text/css'>

  </head>
  <body style="background:#EEE;">
    <div class="container"><div class="row">
      <div class="col-lg-2 col-md-2 hidden-sm hidden-xs"></div>
      <div class="col-md-8 col-xs-12" style="background:#FFF;">
        
      <img class="hidden-lg hidden-md hidden-sm" src="shilpa2.jpg" style="margin:2% 20% 0 20%; width:60%;">
      
      <div class="col-lg-4 col-md-4 col-sm-4 hidden-xs">
        <img class="img-responsive" src="shilpa1.jpg" style="border-radius:15px; margin: 10px 10px 10px 0px;">
      </div>
      <div class="col-lg-8 col-md-8 col-sm-8 col-xs-12">
        <h1>Shilpa Nadimpalli Kobren</h1>
        <hr style="padding-top:0;margin-top:0;">
        <p>I'm a Ph.D. candidate at <a target="_blank" href="http://www.princeton.edu/">Princeton University</a> in the <a target="_blank" href="http://www.cs.princeton.edu/">Department of Computer Science</a> and the <a href="http://www.princeton.edu/genomics/" target="_blank">Lewis-Sigler Institute for Integrative Genomics</a>. My research is in <a target="_blank" href="http://www.cs.princeton.edu/research/areas/compbio">computational and systems biology</a>, where I focus on detecting and interpreting protein interaction and cellular network perturbations within and across organisms in <a target="_blank" href="http://www.cs.princeton.edu/~mona/">Dr. Mona Singh</a>'s lab.</p>
        <p>As an undergraduate at <a href="http://www.tufts.edu/" target="_blank">Tufts University</a>, I worked with <a href="http://www.cs.tufts.edu/~cowen/" target="_blank">Dr. Lenore Cowen</a> on improving methods for protein structural alignments.</p>
        <p>I enjoy traveling, philosophy, tulips, and pumpkin muffins.</p>
        <!-- Here's a link to my hilarious <a href="http://shilpanadimpalli.com/babysite/index.html">first website ever</a>, back by popular demand. -->
        
        <a type="button" class="btn btn-success btn-sm" target="_blank" href="mailto:snadimpa@princeton.edu">Email</a>
        <a type="button" class="btn btn-primary btn-sm" target="_blank" href="http://www.linkedin.com/in/shilpanadimpalli">LinkedIn</a>
        <a type="button" class="btn btn-danger btn-sm" target="_blank" href="ShilpaKobren_cv.pdf">CV (PDF)</a>
        
        <hr class="hidden-md hidden-sm hidden-lg">
        </div>
        <div style="clear:both"></div>
        <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12" style="padding-bottom:3%;"><h3>Publications</h3></div>
        
        <div style="padding:3% 0 3% 0;">
        <div class="col-md-2 col-sm-2 hidden-xs"><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005011" target="_blank"><img class="img-responsive" src="nadimpalli2015b.jpg"></a></div>
        <div class="col-md-10 col-sm-10 col-xs-12">
          <p class="title">Pervasive variation of transcription factor orthologs contributes to regulatory network divergence</p>
          <p class="author">Shilpa Nadimpalli, Anton V. Persikov, and Mona Singh (2015)</p>
          <p class="journal">PLoS Genetics</p>
          <p class="hidden-xs abstract truncate">Differences in transcriptional regulatory networks underlie much of the phenotypic variation observed across organisms. Changes to <i>cis</i>-regulatory elements are widely believed to be the predominant means by which regulatory networks evolve, yet examples of regulatory network divergence due to transcription factor (TF) variation have also been observed. To systematically ascertain the extent to which TFs contribute to regulatory divergence, we analyzed the evolution of the largest class of metazoan TFs, Cys<sub>2</sub>-His<sub>2</sub> zinc finger (C2H2-ZF) TFs, across 12 <i>Drosophila</i> species spanning ~45 million years of evolution. Remarkably, we uncovered that a significant fraction of all C2H2-ZF 1-to-1 orthologs in flies exhibit variations that can affect their DNA-binding specificities. In addition to loss and recruitment of C2H2-ZF domains, we found diverging DNA-contacting residues in ~44% of domains shared between <i>D. melanogaster</i> and the other fly species. These diverging DNA-contacting residues, present in ~70% of the <i>D. melanogaster</i> C2H2-ZF genes in our analysis and corresponding to ~26% of all annotated <i>D. melanogaster</i> TFs, show evidence of functional constraint: they tend to be conserved across phylogenetic clades and evolve more slowly than other diverging residues. These same variations were rarely found as polymorphisms within a population of <i>D. melanogaster</i> flies, indicating their rapid fixation. The predicted specificities of these dynamic domains gradually change across phylogenetic distances, suggesting stepwise evolutionary trajectories for TF divergence. Further, whereas proteins with conserved C2H2-ZF domains are enriched in developmental functions, those with varying domains exhibit no functional enrichments. Our work suggests that a set of highly dynamic and largely unstudied TFs are a likely source of regulatory variation in <i>Drosophila</i> and other metazoans.</p>
          <p class="abstract">
            <a type="button" class="btn btn-warning btn-xs" target="_blank" href="nadimpalli-plosgenet2015.pdf">PDF</a>
            <a type="button" class="btn btn-info btn-xs" href="Nadimpalli2015a.bib">BibTex</a>
            <b>doi:</b> <a href="http://dx.doi.org/10.1371/journal.pgen.1005011" target="_blank">10.1371/journal.pgen.1005011</a>
          </p>
        </div>
        </div>

        <div style="clear:both;"></div>

        <div style="padding-top:3%;">
        <div class="col-md-2 col-sm-2 hidden-xs"><a href="http://www.biomedcentral.com/1471-2105/13/259" target="_blank"><img class="img-responsive" src="daniels2012a.jpg"></a></div>
        <div class="col-md-10 col-sm-10 col-xs-12">
          <p class="title">Formatt: Correcting protein multiple structural alignments by incorporating sequence alignment</p>
          <p class="author">Noah M. Daniels, Shilpa Nadimpalli, and Lenore Cowen (2012)</p>
          <p class="journal">BMC Bioinformatics</p>
          <p class="hidden-xs abstract truncate"><b>Background:</b> The quality of multiple protein structure alignments are usually computed and assessed based on geometric functions of the coordinates of the backbone atoms from the protein chains. These purely geometric methods do not utilize directly protein sequence similarity, and in fact, determining the proper way to incorporate sequence similarity measures into the construction and assessment of protein multiple structure alignments has proved surprisingly difficult.<br/><b>Results:</b> We present Formatt, a multiple structure alignment based on the Matt purely geometric multiple structure alignment program, that also takes into account sequence similarity when constructing alignments. We show that Formatt outperforms Matt and other popular structure alignment programs on the popular HOMSTRAD benchmark. For the SABMark twilight zone benchmark set that captures more remote homology, Formatt and Matt outperform other programs; depending on choice of embedded sequence aligner, Formatt produces either better sequence and structural alignments with a smaller core size than Matt, or similarly sized alignments with better sequence similarity, for a small cost in average RMSD.<br/><b>Conclusions:</b> Considering sequence information as well as purely geometric information seems to improve quality of multiple structure alignments, though defining what constitutes the best alignment when sequence and structural measures would suggest different alignments remains a difficult open question.</p>
          <p class="abstract">
            <a type="button" class="btn btn-warning btn-xs" target="_blank" href="daniels-bmcbio2012.pdf">PDF</a>
            <a type="button" class="btn btn-info btn-xs" href="Daniels2012a.bib">BibTex</a>
            <b>doi:</b> <a href="http://dx.doi.org/10.1186/1471-2105-13-259" target="_blank">10.1186/1471-2105-13-259</a>
          </p>
        </div>
        </div>

        <div style="clear:both;"></div>

        <div style="padding-top:3%;">
        <div class="col-md-2 col-sm-2 hidden-xs"><a href="http://dl.acm.org/citation.cfm?doid=2147805.2147842" target="_blank"><img class="img-responsive" src="nadimpalli2011a.jpg"></a></div>
        <div class="col-md-10 col-sm-10 col-xs-12">
          <p class="title">Formatt: Correcting protein multiple structural alignments by sequence peeking</p>
          <p class="author">Shilpa Nadimpalli*, Noah M. Daniels*, and Lenore Cowen (2011)</p>
          <p class="journal">Proceedings of the 2011 ACM Conference on Bioinformatics, Computational Biology and Biomedicine (BCB)</p>
          <p class="hidden-xs abstract truncate">We present Formatt, a multiple structure alignment program based on the Matt purely geometric multiple structural alignment program, that also takes into account sequence similarity when constructing alignments. We show that Formatt is superior to Matt in alignment quality based on objective measures (most notably Staccato sequence and structure scores) while preserving the same advantages in core length and RMSD that Matt has as a flexible structure aligner, as compared to other multiple structure alignment programs on popular benchmark datasets. Applications include producing better training data for threading methods.</p>
          <p class="abstract">
            <a type="button" class="btn btn-warning btn-xs" target="_blank" href="nadimpalli-acmbcb2011.pdf">PDF</a>
            <a type="button" class="btn btn-info btn-xs" href="Nadimpalli2011a.bib">BibTex</a>
            <b>doi:</b> <a href="http://dx.doi.org/10.1145/2147805.2147842" target="_blank">10.1145/2147805.2147842</a>
          </p>
        </div>
        </div>

        <div style="clear:both;"></div>

        <div style="padding-top:3%;">
        <div class="col-md-2 col-sm-2 hidden-xs"><a href="http://dl.acm.org/citation.cfm?doid=1570256.1570351" target="_blank"><img class="img-responsive" src="rieffel2009a.jpg"></a></div>
        <div class="col-md-10 col-sm-10 col-xs-12">
          <p class="title">Evolving soft robotic locomotion in PhysX</p>
          <p class="author">John Rieffel, Frank Saunders, Shilpa Nadimpalli, Harvey Zhou, Soha Hassoun, Jason Rife, and Barry Trimmer (2009)</p>
          <p class="journal">Proceedings of the 2009 ACM Conference on Genetic and Evolutionary Computation (GECCO)</p>
          <p class="hidden-xs abstract truncate">Given the complexity of the problem, genetic algorithms are one of the more promising methods of discovering control schemes for soft robotics. Since physically embodied evolution is time consuming and expensive, an outstanding challenge lies in developing fast and suitably realistic simulations in which to evolve soft robot gaits. We describe two parallel methods of using NVidia’s PhysX, a hardware-accelerated (GPGPU) physics engine, in order to evolve and optimize soft bodied gaits. The first method involves the evolution of open-loop gaits using a reduced-order lumped parameter model. The second method involves harnessing PhysX’s soft-bodied material simulation capabilites. In each case we discuss the the challenges and possibilities involved in using the PhysX for evolutionary soft robotics.</p>
          <p class="abstract">
            <a type="button" class="btn btn-warning btn-xs" target="_blank" href="rieffel-gecco2009.pdf">PDF</a>
            <a type="button" class="btn btn-info btn-xs" href="Rieffel2009a.bib">BibTex</a>
            <b>doi:</b> <a href="http://dx.doi.org/10.1145/1570256.1570351" target="_blank">10.1145/1570256.1570351</a>
          </p>
        </div>
        </div>

        <div style="clear:both;"></div>
        
      <hr>
      <div class="col-md-4 col-sm-4 col-xs-12">
      <p><h3 class="header">Selected Honors</h3>
        <ul>
          <li><a href="http://www.siebelscholars.com/" target="_blank">Siebel Scholarship</a> (2015-present)</li>
          <li><a href="https://www.nsfgrfp.org/" target="_blank">NSF's Graduate Research Fellowship</a> (2011-present)</li>
          <li>Princeton's Gordon Wu Fellowship (2011-present)</li>
          <li><a href="http://www.google.com/anitaborg/" target="_blank">Google's Anita Borg Memorial Scholarship</a> (2010)</li>
          <li>Featured in Princeton's 2014 <a href="http://giving.princeton.edu/ag" target="_blank">Annual Giving</a> letter to graduate alumni</li>
          <li>Neubauer Scholarship (2007-2011)</li>
          <li>James Schmolze Award for excellence in computer science (2011)</li>
          <li>Class of 1882 Prize Scholarship (2011)</li>
          <li>Audrey Butvay Gruss Award (2010)</li>
        </ul>
      </p>
    </div>
    <div class="col-md-4 col-sm-4 col-xs-12">
      <p><h3 class="header">Teaching</h3>
        <ul>
          <li>TA for Princeton's <a href="http://www.cs.princeton.edu/courses/archive/fall12/cos126/info.html" target="_blank">COS 126, General Computer Science</a> (Fall 2012 and Spring 2013)</li>
          <li>TA for Tufts' <a href="http://www.cs.tufts.edu/comp/40/" target="_blank">CS 40, Machine Structure and Assembly Language Programming</a> (Fall 2009)</li>
          <li>TA for Tufts' <a href="http://www.cs.tufts.edu/comp/105/" target="_blank">CS 80, Programming Languages</a> (Spring 2010)</li>
          <li>Taught high school bioinformatics seminars through an <a href="http://ase.tufts.edu/chemistry/hhmi/" target="_blank">HHMI Arrays Project</a> (Spring 2010)</li>
          <li>Mentored 3 undergraduates through Princeton's <a href="http://www.cs.princeton.edu/academics/ugradpgm/spe/home/" target="_blank">Summer Programming Experiences</a> (Summer 2013)</li>
        </ul></p>
    </div>
    <div class="col-md-4 col-sm-4 col-xs-12">
      <p><h3 class="header">Conferences</h3>
        <ul>
	  <li>Organizing committee of the 2017 <a href="http://acm-bcb.org/" target="_blank">ACM Conference on Bioinformatics, Comp. Bio. and Biomedicine</a>
	  <li>Platform presentation at the 2017 <a href="http://meetings.cshl.edu/home.aspx?name=NYAMQB&year=17" target="_blank">NY Area Meeting in Quantitative Biology</a>
          <li>Poster presentation at the 2015 <a href="http://www.iscb.org/recomb-regsysgen2015" target="_blank">RECOMB/ISCB Conference on Regulatory and Systems Genomics</a></li>
          <li>Poster presentation at the 2014 CSHL meeting on <a href="http://meetings.cshl.edu/meetings/2014/systems14.shtml" target="_blank">Systems Biology: Global Regulation of Gene Expression</a></li>
          <li>Travel grant award recipient for <a href="http://cra-w.org/ArticleDetails/tabid/77/ArticleID/55/Default.aspx" target="_blank">CRA-W's Grad Cohort Workshop</a> in Seattle (Spring 2012)</li>
        </ul>
      </p>
    </div>
    <div style="clear:both;"></div>
    <hr>
    <div class="col-md-8 col-sm-8 col-xs-12">
      <p><h3 class="header">Other Activities</h3>
        <span style="font-size:.85em;line-height:.85em;">Most recently, I've served as a student activity co-chair for <a href="http://acm-bcb.org/">ACM BCB 2017</a> in Boston. I've also served as the Computer Science Representative and an Events Board member on <a href="https://gsg.princeton.edu/" target="_blank">Princeton's Graduate Student Government</a> (2011-2014), as President of <a href="http://www.princeton.edu/hillel/student-life/grads-young-professionals/" target="_blank">Jewish Graduate Students and Young Professionals</a> (2012-2014), and as a member of <a href="http://www.princeton.edu/~gwise/" target="_blank">Princeton's Graduate Women in Science and Engineering</a> (2011-present). I am a college interviewer through <a href="http://admissions.tufts.edu/taap" target="_blank">Tufts' Alumni Admissions Program</a>. In 2013 and 2015, I served as a panelist for two undergraduate events: "How to Get the Most Out of Office Hours" and "Applying to Graduate School in Computer Science". I have also been selected to meet with the Princeton Leadership Council, Princeton Board of Trustees, and Princeton Alumni Association to talk about programs and initiatives in the <a href="http://www.princeton.edu/engineering/" target="_blank">School of Engineering</a>.</span></p>
      </div>
      <div class="col-md-4 col-sm-4 col-xs-12">
      <p>For more information, please see my <a href="ShilpaKobren_cv.pdf" target="_blank">CV</a>.</p>
      <div class="hidden-xs"><a href="ShilpaKobren_cv.pdf" target="_blank"><img src="ShilpaKobren_cv.jpg" style="width:80%;margin:0 10% 0 10%;" /></a></div>
      </div>  
        <div style="clear:both;"></div>
        
      <hr>
      <p class="abstract" style="padding:5px 0 5px 0;">Last updated: 17-Apr-2017 | Created with <a href="http://getbootstrap.com/" target="_blank">Bootstrap</a></p>
      
      </div>
      <div class="col-md-2 hidden-xs"></div>
    </div></div>

  </body>
</html>
